upload-script/active_learning.py

```diff
@@ -12,7 +12,8 @@
 from rastervision.pytorch_learner import *
 from rastervision.pytorch_backend.examples.utils import read_stac
 
-image_uri = '../jacksonville.sub.tif'
+train_image_uri = './jacksonville.sub.tif'
+val_image_uri = '/mnt/data/workshop/materials/clipped-jacksonville-2016.tif'
```

Comment:
let's make these both relative, and also the training scene is still up a dir

```suggestion
train_image_uri = '../jacksonville.sub.tif'
val_image_uri = '../clipped-jacksonville-2016.tif'
```